create-manifest.js

```diff
@@ -0,0 +1,12 @@
+module.exports = function createWebappManifest(details) {
+    let manifestObj =  {
+        "name": details.name,
+        "short_name": details.name,
```

Comment:
If the name is multi word can we shrink it into smaller name 

`The name (maximum of 45 characters)`

`The short_name (maximum of 12 characters recommended) `

```json
  "manifest_version": 2,
  "version": "versionString",
```

can we add these too ?

Reply:
I have exams till Saturday. Would it be alright if I submit the patch incorporating these changes on Saturday?

Reply:
How to shrink the name? Do I remove whitespaces in between the words or just simply take the first 45 characters of the string?

Reply:
use `validate` function in `inquirer`

Reply:
Got It